### ISSUE TITLE
Npub?

### ISSUE THREAD

**Issue by lez** [guaka/nostrhitch#4]
Hello!
What's your npub?
I can't find you on Nostr, but want to follow nostrhitch.

**Comment by guaka** [comment]
hitchmap: npub12vz4acq3a94qpfc9kwp98wwtces5ej7n0h6d44pwc6dmucyvggyses5uyk

hitchwiki: npub1zmd4ydxpmkqg9ztm6tfph0kyhqz36txs8cjtsxd22geqwl2y8k5s7x6qpm 
which is coming from https://github.com/Hitchwiki/hwpybot

Both of these are sending out kind 30397 events to relay.trustroots.org, not sure if you can really follow this in other apps. Happy to hear about your experience with this.
Can consider sending out kind 1 notes, to make this hitch stuff visible in the nostr social media sphere... What do you think?

**Comment by guaka** [comment]
If you want to see these notes on a map, the [nostroots expo app](https://github.com/Trustroots/nostroots) currently looks like this:

<img width="447" alt="image" src="https://github.com/user-attachments/assets/4bda9d67-82e5-4805-968b-602ac2f845ab">

**Comment by lez** [comment]
> hitchmap: npub12vz4acq3a94qpfc9kwp98wwtces5ej7n0h6d44pwc6dmucyvggyses5uyk
> 
> hitchwiki: npub1zmd4ydxpmkqg9ztm6tfph0kyhqz36txs8cjtsxd22geqwl2y8k5s7x6qpm which is coming from https://github.com/Hitchwiki/hwpybot
> 
> Both of these are sending out kind 30399 events to relay.trustroots.org, not sure if you can really follow this in other apps. Happy to hear about your experience with this. Can consider sending out kind 1 notes, to make this hitch stuff visible in the nostr social media sphere... What do you think?

I think you can come up with an appropriate kind 1 event in some situations. For example, if I decide to hitch from Budapest to Rome, you can compose a message, tagging all major cities on the way, with a message editable by the user something along the lines of "hey! I'm passing #slovenia, #venice, #bologna on the way to #rome next week! DM me if you want to bump up, have a coffee and share our nostr visions! I'm planning the trip right now on #nostrhitch!"

This is neither too spammy on the social level, and useful to the users.

Could not find your profiles, contact me on npub1elta7cneng3w8p9y4dw633qzdjr4kyvaparuyuttyrx6e8xp7xnq32cume pls. I still have a trustroot sticker on my wall that you gave to me in Bitcoin Atlantis. :)

**Comment by guaka** [comment]
Yeah I'd like to see messages like that, but nostrhitch is currently just a nightly script pulling in data from hitchmap.com. Hitchmap doesn't have any form of login currently.
